charts/ColorScale.ts

```diff
@@ -130,32 +139,28 @@ export class ColorScale {
     }
 
     @computed get colorScheme(): ColorScheme {
-        const colorScheme = ColorSchemes[this.baseColorScheme]
-        return colorScheme !== undefined ? colorScheme : this.defaultColorScheme
+        return ColorSchemes[this.baseColorScheme] ?? this.defaultColorScheme
     }
 
     @computed get singleColorScale(): boolean {
         return this.colorScheme.singleColorScale
     }
 
     @computed get autoMinBinValue(): number {
-        const minValue = Math.min(0, this.valuesWithoutOutliers[0])
-        const magnitude = Math.floor(Math.log(minValue) / Math.log(10))
-        return Math.min(0, round(minValue, -magnitude))
+        const minValue = Math.min(0, this.sortedNumericValuesWithoutOutliers[0])
```

Comment:
if `sortedNumericValuesWithoutOutliers` is an empty array, `Math.min(0, undefined)` will return `NaN` which is nasty, so better handle that case if it ever comes up